metricbeat/scripts/mage/docsCollector.go

```diff
@@ -0,0 +1,488 @@
+package mage
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/elastic/beats/dev-tools/mage"
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+)
+
+// this struct provides module-level data that will be used to populate the module list
+type moduleData struct {
+	Path       string
+	Base       string
+	Title      string `yaml:"title"`
+	Release    string `yaml:"release"`
+	Dashboards bool
+	Settings   []string `yaml:"settings"`
+	CfgFile    string
+	Asciidoc   string
+	Metricsets []metricsetData
+}
+
+//check to see if the config list has a particular item
```

Comment:
And have an space after the comment marks :slightly_smiling_face: 
```suggestion
// check to see if the config list has a particular item
```

Reply:
Bah, I thought I got all of those!